src/Terrain/TerrainQuery.cc

```diff
@@ -901,6 +910,19 @@ QList<double> UnitTestTerrainQuery::_requestCoordinateHeights(const QList<QGeoCo
             long dx = regionSizeDeg/one_second_deg;
             double fraction = 1.0 * x / dx;
             result.append(std::round(UnitTestTerrainQuery::LinearSlopeRegion::minAMSLElevation + (fraction * UnitTestTerrainQuery::LinearSlopeRegion::totalElevationChange)));
+        } else if (hillRegion.contains(coordinate)) {
+            double arc_second_meters = (6371000. / 3600.) * (M_PI / 180);
```

Comment:
What is `6371000` ? Please create an intermediary variable to describe this magic number.